section/DrawFunc.cpp

```diff
@@ -116,6 +116,10 @@ namespace Moho
             return t;
         }
 
+        void ReleaseTexture(Texture *t) {
+            asm("call 0x004260B0;" : : [t] "a"(t) : "edx", "ecx");
```

Comment:
Where did you get these clobbers from?

Reply:
These registers are being altered in this function.